agent/ansible/pbench/agent/roles/pbench_agent_files_install/README.md

```diff
@@ -0,0 +1,53 @@
+pbench_agent_files_install
+==========================
+
+This role is used by pbench_agent_config to install a pbench-agent.cfg
+file and an ssh key. pbench_agent_config sets the "source" and "files"
+parameters to the URL of the containing directory and the list of
+files to copy resp. and calls this role as a subroutine. This role
+copies the files to local storage. We do it that way so that we can
+copy the files to systems outside the firewall that do not have access
```

Comment:
s/the firewall/a firewall/
s/that dot not have access to the (internal) URLs/that do not have access to the provided URLs/

Reply:
Will fix.